lib/round.rb

```diff
@@ -0,0 +1,46 @@
+class Round 
+  attr_accessor :deck, 
+                :turns, 
+                :current_card, 
+                :number_correct, 
+                :take_turn,    
```

Comment:
Like `current_card`, `take_turn` is a method you are defining elsewhere in the class so we should remove it from our `attr_readers` to avoid redundancy and conflict.